web/public/templates/dojos/events/manage-dojo-events.dust

```diff
@@ -24,13 +24,19 @@
                 <tr ng-repeat="event in events">
                   <td class="pointer"><a ui-sref="manage-applications({dojoId: dojoId, eventId:event.id})">{{ event.name }}</a></td>
                   <td>{{ event.formattedDate }}</td>
-                  <td>{{ event.eventStats.capacity }}</td>
-                  <td>{{ event.eventStats.totalApplicants }}</td>
-                  <td>{{ event.eventStats.totalAttending }}</td>
+                  <td><span ng-show="!event.eventbriteId">{{ event.eventStats.capacity }}</span></td>
```

Comment:
What appears otherwise? `undefined` or something like that? Also, wouldn't `ng-if` be better here, since it's render once and forget?